lib/Graph/Graph.cpp

```diff
@@ -1285,13 +1285,23 @@ SwishNode *Function::createSwish(llvm::StringRef name, NodeValue input,
   return createSwish(name, OT, input);
 }
 
-HardSigmoidNode *Function::createHardSigmoid(llvm::StringRef name,
+ClipNode *Function::createHardSigmoid(llvm::StringRef name,
```

Comment:
@jfix71 
As you mentioned, I stopped creating a new node, instead create `ClipNode`.
Is this what you expect?

Reply:
Changing this part made me think that if we change such way, maybe other operators like ReLU better changed likewise (I mean perhaps we need a consistent policy on which operator should be lowered and which one should not, in this currently supported operators).